src/main/java/stsjorbsmod/JorbsMod.java

```diff
@@ -91,6 +91,9 @@
         // Use on a card that remembers a memory, which is mechanic specific to the Wanderer character.
         @SpireEnum(name = "REMEMBER_MEMORY")
         public static AbstractCard.CardTags REMEMBER_MEMORY;
+
+        // Use on a card which Exerts the Culler
+        public static AbstractCard.CardTags HAS_EXERT;
```

Comment:
#474 is probably going to end up using a `SpireField` rather than a tag for exert; I think I would probably recommend just leaving out the exert check from here for now, and if this PR ends up going in before #474, file an issue or a comment on #474 to track adding the exert check once exert is done